CHANGELOG.md

```diff
@@ -4,6 +4,9 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [unreleased]
 
+- Fix: Replace `k-u-small` by `k-u-size-micro` and `k-u-strong` by
```

Comment:
En lisant cette ligne j'ai l'impression qu'on le remplace dans les utilitaires. Je noterais plus un truc du style :

```md
// changelog.md

- Fix: Use `k-u-weight-regular` instead of `k-u-strong` in `ProjectCard` component.
```

```md
// karl_changelog.md

- Fix: Use `k-u-size-micro` instead of `k-u-small` and `k-u-weight-regular` instead of `k-u-strong` in examples.
```

Qu'en penses-tu ?